app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -1195,6 +1189,11 @@ public void startDescriptionEditActivity(@Nullable String text) {
         }
     }
 
+    private void startTalkTopicActivity(@NonNull PageTitle pageTitle) {
+        // If we're currently looking at a User page, then go directly to the corresponding User Talk page.
```

Comment:
This comment is no longer relevant here.

Reply:
Done.